Mathlib/Algebra/Module/LocalizedModule.lean

```diff
@@ -1082,6 +1085,83 @@ theorem mkOfAlgebra {R S S' : Type _} [CommRing R] [CommRing S] [CommRing S'] [A
 
 end Algebra
 
+variable [Module (Localization S) M'] [IsScalarTower R (Localization S) M']
+
+/--
+- If `(f : M →ₗ[R] M')` is a localization of modules, then the map
+- `(localization S) × M → N, (s, m) ↦ s • f m` is a tensor product.
```

Comment:
> What does it mean for a bilinear map to be a tensor product? Did you mean to say this? And what is `N`?

_Originally posted by @kbuzzard in https://github.com/leanprover-community/mathlib/pull/17973#discussion_r1256596031_

--
oops, `N` here should be `M'`
By "is a tensor product", I mean that this map (along with `M'`) is universal.
Both the binary map and object are necessary to define the tensor product, so I don't have an issue calling either "the tensor product". This also goes well with how our typeclasses are defined. (`IsBaseChange` is defined as `IsTensorProduct` for the map given in this docstring).